pyrevitlib/rpw/db/family.py

```diff
@@ -93,7 +94,9 @@ def in_assembly(self):
         Returns:
             (bool): True if element is inside an AssemblyInstance
         """
-        if self._revit_object.AssemblyInstanceId.IntegerValue == -1:
+        value_func = get_value_func()
+        assembly_id_value = value_func(self._revit_object.AssemblyInstanceId)
+        if assembly_id_value == -1:
             return False
         else:
             return True
```

Comment:
we can just `return assembly_id_value != -1`, or even `return value_func(self._revit_object.AssemblyInstanceId) != -1`

Reply:
I totally agree with you:
- not in scope 😄 
- while I am at it.

I fixed a few things here and there but it feels like a never ending story to dig further for now
I thought also about adding docstrings as well ;p